src/components/modal/modal.spec.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-unused-expressions, max-nested-callbacks */
```

Comment:
if you use chaijs expectations you might want to write:
```js
expect('Rouven').to.be.ok
expect(['Gian', 'Luc', 'a'].to.have.a.length
```

In this case eslint complains because these chains are not called explicitly :(